sovtokenfees/sovtokenfees/static_fee_req_handler.py

```diff
@@ -128,6 +132,27 @@ def doStaticValidation(self, request: Request):
         else:
             super().doStaticValidation(request)
 
+    def _fees_specific_validation(self, request: Request):
+        operation = request.operation
+        current_fees = self._get_fees()
+        constraint = self.get_auth_constraint(operation)
+        wrong_aliases = []
+        self._validate_metadata(self.fees, constraint, wrong_aliases)
+        if len(wrong_aliases) > 0:
+            raise InvalidClientMessageException(request.identifier,
+                                                request.reqId,
+                                                "Fees alias(es) {} does not exist in current fees {}".format(", ".join(wrong_aliases),
```

Comment:
Please extend the error message with the following: `Please add the aliases via SET_FEES transaction first`.